### DISCUSSION TITLE
View images in QuPath Application

### DISCUSSION THREAD

**Discussion by Suchi97** [Project-MONAI/MONAILabel#840] [Q&A]
Hi Team,

We are trying to use the QuPath along with MONAI Label

We have inserted url in the Monai Label path and the connection seems sucessfull.

![image](https://user-images.githubusercontent.com/25792114/174119996-ac493d87-f578-4504-bb0f-1ee38e23a402.png)

So when we try to open a new image it always goes on to the local machine, but the data is on remote PC, how can we access the data from the remote pc as we do in 3D slicer

Any help is appreciated.
Thanks in advance

**Comment by SachidanandAlle** [comment]
Currently it's limited.. pathology images are very large.. so we assume they are available at your local system where qupath is running.. you open the same file (with same name) which exists in both the sides..

**Reply by Suchi97** [reply]
Thanks @SachidanandAlle for your reply.
